tablesorter-grouping.js

```diff
@@ -95,6 +99,30 @@
         currentRow.removeClass(zebraClasses).addClass(zebraClass);
       }
     });
+
+    var rowGroupIndex = 0;
+    var previousRow;
+
+    //Apply table group numbers
+    $("tr", table).each(function(idx) {
+      var currentRow = $(this);
+
+      if(previousRow === undefined) previousRow = currentRow;
+        
+      //Add the class to the first row in the group
+      if(currentRow.hasClass(groupedRowClass) && !previousRow.hasClass(groupedRowClass))
+        previousRow.addClass(groupNumClass+rowGroupIndex);
+    
+      //Add the correct group number to the grouped rows
+      if(currentRow.hasClass(groupedRowClass))
+        currentRow.addClass(groupNumClass+rowGroupIndex);
+
+     //Decides whether its at the end of a group and increments the index
+      if(previousRow.hasClass(groupedRowClass) && !currentRow.hasClass(groupedRowClass))
+          rowGroupIndex++;
+
+      previousRow = currentRow;
+    });
```

Comment:
Instead of storing it in the class information, let's store it in data. That'll just be simpler. It won't be accessible for CSS modifications, but that's probably okay.